### PR TITLE
Added HiBit Startup Manager version 2.3.20.100

### LINKED ISSUE #65007: [Package Request]: Hibit.StartUpManager
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Hibit
Publisher Url: https://www.hibitsoft.ir/
Package Name: Hibit.StartUpManager
Description: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and conrext menu.
Package Version: 2.3.20
Installer URL: https://www.hibitsoft.ir/HibitStartup/HiBitStartupManager-setup-2.3.20.exe

Current Package Identifier (for new versions): 
https://www.hibitsoft.ir/HibitStartup/HiBitStartupManager-setup-2.3.20.exe

### PR DESCRIPTION
**Pull request by licon4812**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65447)

closes https://github.com/microsoft/winget-pkgs/issues/65007

### CODE REVIEW

**Comment by licon4812** [comment]
Merging this should close this: #65007

**Review by Trenly — CHANGES_REQUESTED** [review]
Publisher already exists under a different name; Package ID should be updated to match

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /VERYSILENT
+  Custom: /NORESTART
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml:11]:
```suggestion
InstallerType: inno
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://www.hibitsoft.ir/HibitStartup/HiBitStartupManager-setup-2.3.20.exe
+  InstallerSha256: 1E86E69E3F82982F85A63EBF9DF51FFB20F5018BFA8BE2164DEB0E9274F66A46
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml:16]:
```suggestion
- Architecture: x86
  InstallerUrl: https://www.hibitsoft.ir/HibitStartup/HiBitStartupManager-setup-2.3.20.exe
  InstallerSha256: 1E86E69E3F82982F85A63EBF9DF51FFB20F5018BFA8BE2164DEB0E9274F66A46
- Architecture: x64
  InstallerUrl: https://www.hibitsoft.ir/HibitStartup/HiBitStartupManager-setup-2.3.20.exe
  InstallerSha256: 1E86E69E3F82982F85A63EBF9DF51FFB20F5018BFA8BE2164DEB0E9274F66A46
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: HiBit Startup Manager version 2.3.20.100
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:12]:
```suggestion
PackageName: HiBit Startup Manager
```
Versions should be kept out of package names

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# PackageUrl:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:13]:
```suggestion
PackageUrl: http://www.hibitsoft.ir/StartupManager.html
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.hibitsoft.ir/
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# LicenseUrl: 
+Copyright: Copyright @ 2017-2022 HiBitSoft.IR All Rights Reserved
+# CopyrightUrl: 
+ShortDescription: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and conrext menu.
+# Description: 
+# Moniker: 
+# Tags: 
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:24]:
```suggestion
ReleaseNotes: |-
  Added Slovenian language.
  Improved GUI and fixed all known bugs.
ReleaseNotesUrl: http://www.hibitsoft.ir/HibitStartup/Changelog.txt
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /VERYSILENT
+  Custom: /NORESTART
+UpgradeBehavior: uninstallPrevious
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.installer.yaml:12]:
```suggestion
UpgradeBehavior: uninstallPrevious
ReleaseDate: 2022-01-25
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl: 
+# PublisherSupportUrl:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:9]:
```suggestion
PublisherSupportUrl: https://www.hibitsoft.ir/About.html
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:11]:
```suggestion
Author: Mohammed Raee
```

File: manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+# PrivacyUrl: 
+# Author: 
+PackageName: HiBit Startup Manager version 2.3.20.100
+# PackageUrl: 
+License: Freeware
+# LicenseUrl: 
+Copyright: Copyright @ 2017-2022 HiBitSoft.IR All Rights Reserved
+# CopyrightUrl: 
+ShortDescription: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and conrext menu.
+# Description:
```
Comment (Trenly) [inline manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml:19]:
```suggestion
ShortDescription: Manage the List of All Applications that are Loaded Automatically When Windows Starts Up.
Description: HiBit Startup Manager is a powerful Startup utility that Manage autostart programs by adding or removing apps, editing their properties and also Manage process, services, task scheduler and context menu.
```
Short description from the website helps correlation

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**